lib/rack/reloader.rb

```diff
@@ -24,7 +22,7 @@ module Rack
   # It is performing a check/reload cycle at the start of every request, but
   # also respects a cool down time, during which nothing will be done.
   class Reloader
-    using ::Rack::RegexpExtensions if RUBY_VERSION < '2.4'
+    (require_relative 'core_ext/regexp'; using ::Rack::RegexpExtensions) if RUBY_VERSION < '2.4'
```

Comment:
For the next release of Rack, why don't we drop < 2.4? @tenderlove @jeremyevans

Reply:
Not sure when we plan to release Rack 2.2, but that sounds OK to me.